assets/src/stories-editor/components/custom-video-block-edit.js

```diff
@@ -134,7 +134,9 @@ const CustomVideoBlockEdit = ( { instanceId, isSelected, className, attributes,
 		if ( src && ! isBlobURL( src ) ) {
 			getContentLengthFromUrl( src ).then( setVideoSize );
 		}
-	}, [ allowedVideoMimeTypes, id, mediaUpload, noticeOperations, setAttributes, src ] );
+		// Disable reason: src is constantly changing, but we want this hook to fire only once (like componentDidMount).
```

Comment:
To ensure running only once, wouldn't we need to pass just `[]` as the second argument?

Reply:
Fixed in c6ec90ad63545febfa973d134d9e74b84389a996